### PR TITLE
# 184948162 seller should get products in his/her collection

### PR DESCRIPTION
**Pull request by Patienceineza**
#### What does this PR do?

this PR is for  seller to be able to view their own collection as well as getting one product from their collection

#### Description of Task to be completed?

-this task includes to create UI for both view all products in collection as well as  get one product from collection
-fetch APIs for those endpoints

#### How should this be manually tested?

-first login and got to _avatar_ of the user and click on  _your collection_
-as a seller you will see your profile information and all products in your collection
- click on any product if you want to get more details about it

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184948162

#### Screenshots (if appropriate)

Screenshot 2023-05-02 at 18 40 13
Screenshot 2023-05-02 at 18 41 40

Screenshot 2023-05-07 at 18 28 16

### CODE REVIEW

**Review by Habihirwe — APPROVED** [review]
well done

**Review by Habihirwe — APPROVED** [review]
well done

**Review by Habihirwe — APPROVED** [review]
well done

File: src/Views/Home.jsx [hunk 1]
```diff
@@ -7,7 +7,15 @@ function Home() {
 
   return (
     <>
-      <h1>Homepage</h1>
+      <h1>
+        {' '}
+        <Link to="/collection" data-testid="signup">
```
Comment (blackd44) [inline src/Views/Home.jsx:12]: can you change these end paths
cause are not collects

**Review by blackd44 — APPROVED** [review]
You have 2 commits 
other are fine

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Resolve the conversations

**Review by Martha-Iradukunda — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**